ch00python/015variables.ipynb

```diff
@@ -921,7 +858,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If I now go back to Input 33, reading `number = number + 1`, and run it again, with shift-enter. Number will change from 2 to 3, then from 3 to 4. Try it!"
+    "If I now go back to Input 31, reading `number = number + 1`, and run it again, with shift-enter. Number will change from 2 to 3, then from 3 to 4. Try it!"
```

Comment:
This sentence's structure is not quite correct. It's also not completely clear what is being updated (the variable) and what isn't (the output).

```suggestion
    "If I now go back to Input 31, reading `number = number + 1`, I can run it again, with Shift-Enter. The value of `number` will change from 2 to 3, then from 3 to 4 - but the output of the next cell (containing the `print` statement) will not change unless I rerun that too. Try it!"
```